src/MorphoTokenEthereum.sol

```diff
@@ -27,12 +27,10 @@ contract MorphoTokenEthereum is DelegationToken {
     /// @param owner The new owner.
     /// @param wrapper The wrapper contract address to migrate legacy MORPHO tokens to the new one.
     function initialize(address owner, address wrapper) external initializer {
-        require(owner != address(0), ZeroAddress());
```

Comment:
The error can be removed I think

Reply:
wait just seeing [this](https://github.com/morpho-org/morpho-token-upgradeable/pull/70#discussion_r1819398451) but i can't find the changes